src/tls13.c

```diff
@@ -3184,21 +3184,26 @@ static int DoTls13CertificateRequest(WOLFSSL* ssl, const byte* input,
     }
     *inOutIdx += len;
 
+#ifndef WOLFSSL_NO_CLIENT_CERT_ERROR
```

Comment:
Having this controlled with a define at build time only seems cumbersome.  Is this really the most flexible / easy solution for users?

Reply:
It could be an API but I don't think it is something that should be used.
If as a client you don't have a certificate then the specification says to return an empty certificate message.
But, if the customer really wants to fail in the client because they didn't set a certificate, then they can compile this in.

Reply:
Is this forcing INVALID_PARAMETER as the return code?  If so, can we be more specific on the error and have a message?

Reply:
INVALID_PARAMETER is the right error I believe - couldn't match the algorithm requirements for a certificate.

Reworked to explicitly return INVALID_PARAMETER and WOLFSSL_MSG added.

Reply:
Can you add a NO_CERT_ERROR as the return instead?  I can see a support issue if this happens in the future and trying to determine whether it's one of 20 INVALID_PARM errors or actually no client cert loaded (especially if in the field and logging isn't available).

Reply:
Changed as requested.